### DISCUSSION TITLE
Correct way to uninstall Mac client using terminal?

### DISCUSSION THREAD

**Discussion by ghost** [amidaware/tacticalrmm#1607] [General]
I have some Mac clients that won't uninstall via the GUI so I need to remove them using terminal.

I could just nuke the rmmagent and tactical.plist but is there a better way to do it?

**Comment by wh1te909** [comment]
https://github.com/amidaware/tacticalrmm/blob/develop/api/tacticalrmm/core/mac_uninstall.sh
